### PR TITLE
[SPARK-21803] [TEST] Remove the HiveDDLCommandSuite

### PR DESCRIPTION
**Pull request by gatorsmile**
## What changes were proposed in this pull request?
We do not have any Hive-specific parser. It does not make sense to keep a parser-specific test suite `HiveDDLCommandSuite.scala` in the Hive package. This PR is to remove it.  

## How was this patch tested?
N/A

### CODE REVIEW

**Comment by gatorsmile** [comment]
cc @cloud-fan

File: sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandSuite.scala [hunk 1]
```diff
@@ -22,19 +22,26 @@ import java.util.Locale
...
 import org.apache.spark.sql.execution.SparkSqlParser
 import org.apache.spark.sql.execution.datasources.CreateTable
 import org.apache.spark.sql.internal.{HiveSerDe, SQLConf}
+import org.apache.spark.sql.test.SharedSQLContext
 import org.apache.spark.sql.types.{IntegerType, StringType, StructField, StructType}
 
 
 // TODO: merge this with DDLSuite (SPARK-14441)
-class DDLCommandSuite extends PlanTest {
+class DDLCommandSuite extends PlanTest with SharedSQLContext {
```
Comment (cloud-fan) [inline sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandSuite.scala]: shall we rename it to `DDLParserSuite`?

**Comment by cloud-fan** [comment]
LGTM

File: sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandSuite.scala (hunk 1 above)
Comment (gatorsmile) [inline sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandSuite.scala]: Sure.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Pull request closed without merging**